.github/workflows/_ios-build-test.yml

```diff
@@ -100,11 +93,11 @@ jobs:
       - name: Build PyTorch Mobile Runtime
         run: |
           # shellcheck disable=SC1091
-          source "${RUNNER_TEMP}/anaconda/bin/activate"
+          source "${MINICONDA_INSTALL_PATH}/bin/activate" "${RUNNER_TEMP}/${CONDA_ENV}"
```

Comment:
You can just use ${CONDA_RUN} on line 100 and remove:

* `source`  on line 96
* export CMAKE_PREFIX_PATH on line 99 

Both of them has been made available by setup-miniconda

Reply:
Fixed.